src/software/thunderscope/robot_diagnostics/drive_and_dribbler_widget.py

```diff
@@ -7,6 +8,93 @@
 from software.thunderscope.proto_unix_io import ProtoUnixIO
 from software.thunderscope.common import common_widgets
 
+import evdev
+from evdev import ecodes
+
+
+class KeyboardMagic:
+    def __init__(self, drive_and_dribble_widget):
+        self.drive_and_dribble_widget = drive_and_dribble_widget
+        self.device = evdev.InputDevice("/dev/input/event3")
```

Comment:
this will be different across machines. The argument to `InputDevice` should be a parameter that can be passed into the constructor